rcgen/src/lib.rs

```diff
@@ -151,13 +151,31 @@ const ENCODE_CONFIG: pem::EncodeConfig = {
 #[non_exhaustive]
 /// The type of subject alt name
 pub enum SanType {
+	OtherName((Vec<u64>, OtherNameValue)),
 	/// Also known as E-Mail address
 	Rfc822Name(Ia5String),
 	DnsName(Ia5String),
 	URI(Ia5String),
 	IpAddress(IpAddr),
 }
 
+/// An other name entry
```

Comment:
Add some context here from your PR description maybe? Like links to the RFC and context on the choice of the supported type(s).

Reply:
Good idea, I've added some context and links